templates/keystoneapi/config/keystone.conf

```diff
@@ -22,7 +22,7 @@ enforce_scope = {{ .enableSecureRBAC }}
 
 [fernet_tokens]
 key_repository=/etc/keystone/fernet-keys
-max_active_keys=2
+max_active_keys={{ .fernetMaxActiveKeys }}
```

Comment:
This setting is only used for Fernet key rotation [1], which we implemented in go. I guess we can keep your change, to be consistent...

[1] https://github.com/openstack/keystone/blob/6433f1ae1c094e58216360d191a21d956a80f658/keystone/common/fernet_utils.py#L261